content/pt/docs/concepts/architecture/controller.md

```diff
@@ -0,0 +1,162 @@
+---
+title: Controladores
+content_template: templates/concept
+weight: 30
+---
+
+{{% capture overview %}}
+
+Em robótica e automação um _control loop_, ou em português _ciclo de controle_, é
+um ciclo não terminado que regula o estado de um sistema.
+
+Um exemplo de ciclo de controlo é um termostato de uma sala.
+
+Quando você define a temperatura, isso indica ao termostato
+sobre o seu *estado desejado*. A temperatura ambiente real é o
+*estado atual*. O termostato atua de forma a trazer o estado atual
+mais perto do estado desejado, ligando ou desligando o equipamento.
+
+{{< glossary_definition term_id="controller" length="short">}}
+
+{{% /capture %}}
+
+
+{{% capture body %}}
+
+## Padrão de controlo (Controller pattern)
+
+Um controlador rastreia pelo menos um tipo de recurso Kubernetes.
+Estes [objetos](/docs/concepts/overview/working-with-objects/kubernetes-objects/#kubernetes-objects)
+têm um campo *spec* (especificação) que representa o *estado desejado*.
+Os controladores para esse recurso são responsáveis por trazer o *estado atual*
+mais perto do *estado desejado*.
+
+O controlador pode executar uma ação ele próprio, ou,
+o que é mais comum, enviar uma mensagem para o Kubernetes
+{{< glossary_tooltip text="API server" term_id="kube-apiserver" >}} que tem
+efeitos colaterais úteis. Você vai ver exemplos disto abaixo.
+
+{{< comment >}}
```

Comment:
I believe this comment was removed on the last version, can you please confirm? Tks.